CustomCraftSML/Serialization/Entries/CustomFood.cs

```diff
@@ -0,0 +1,414 @@
+﻿using System.IO;
+using IOPath = System.IO.Path;
+using CustomCraft2SML.Interfaces.InternalUse;
+using CustomCraft2SML.Serialization.Components;
+using SMLHelper.V2.Crafting;
+using SMLHelper.V2.Handlers;
+using SMLHelper.V2.Utility;
+
+namespace CustomCraft2SML.Serialization.Entries
+{
+    using Common;
+    using Common.EasyMarkup;
+    using CustomCraft2SML.Interfaces;
+    using CustomCraft2SML.PublicAPI;
+    using CustomCraft2SML.Serialization.Lists;
+    using CustomCraft2SML.SMLHelperItems;
+    using System;
+    using System.Collections.Generic;
+
+    internal class CustomFood : EmTechTyped, ICustomFood, ICustomCraft
```

Comment:
`CustomFood` doesn't appear to be fully implementing [ICustomCraft](https://github.com/PrimeSonic/PrimeSonicSubnauticaMods/blob/master/CustomCraftSML/Interfaces/InternalUse/ICustomCraft.cs).  

It needs the `PassesPreValidation();` method.